app/include/dt-bindings/zmk/keys.h

```diff
@@ -1420,3 +1420,68 @@
 #define C_KEYBOARD_INPUT_ASSIST_CANCEL                                                             \
     (HID_USAGE(HID_USAGE_CONSUMER, HID_USAGE_CONSUMER_KEYBOARD_INPUT_ASSIST_CANCEL))
 #define C_KBIA_CANCEL (C_KEYBOARD_INPUT_ASSIST_CANCEL)
+
+#define PLV_N1 (HID_USAGE(HID_USAGE_PLOVER, 0))
```

Comment:
Ideally, we could have defines for the individual usage IDs, in addition to the "full usage" defines you added here.

Reply:
I'm not sure that I follow you here, do you mean that we should have things like:

`#define PLV_0 (HID_USAGE(HID_USAGE_VENDOR_PLOVER, 0))` or do you mean something else?